src/volunteers.html

```diff
@@ -1,253 +1,84 @@
-            </li>
-            <li>
-              <a href="events.html" class="menu-item">Events</a>
-            </li>
-            <li>
-              <a href="https://blog.womenhackfornonprofits.com/" class="menu-item">Blog</a>
-            </li>
-            <li>
-              <a href="contact.html" class="menu-item">Contact</a>
-            </li>
-          </ul>
-        </nav>
-        <!-- Desktop Nav -->
-        <nav id="primary-nav">
-          <ul>
-            <li>
-              <a href="about.html">About</a>
-            </li>
-            <li>
-              <a href="volunteers.html">Volunteers</a>
-            </li>
-            <li>
-              <a href="non-profits.html">Non-Profits</a>
-            </li>
-            <li>
-              <a href="projects.html">Projects</a>
-            </li>
-            <li>
-              <a href="events.html">Events</a>
-            </li>
-            <li>
-              <a href="https://blog.womenhackfornonprofits.com/">Blog</a>
-            </li>
-            <li>
-              <a href="contact.html">Contact</a>
-            </li>
-          </ul>
-        </nav>
+        <h2 class="section-heading">1. Contact Us</h2>
+        <p>Fill in the <a class="text-link" href="volunteer-contact.html">volunteer form</a> and hit send.</p>
       </div>
-    </header>
-    <section class="smaller-banner volunteers-image">
-      <div class="content">
-        <h1 class="title-heading title-cta">Volunteers</h1>
-        <a href="volunteer-contact.html" class="button">Volunteer with us</a>
-      </div>
-    </section>
-    <section>
-      <div class="content">
-        <h2 class="section-heading"> How it works </h2>
-        <div class="col-wrap">
-          <div class="col-3 centered-text">
-            <div class="">
-              <img class="non-profit-icons" src="img/icons/email-us.svg" alt="Email Us Image"></a>
-            </div>
-            <h2 class="section-heading">1. Contact Us</h2>
-            <p>Fill in the <a class="text-link" href="volunteer-contact.html">volunteer form</a> and hit send.</p>
-          </div>
-          <div class="col-3">
-            <div class="">
-              <img class="non-profit-icons" src="img/icons/email-confirmation.svg" alt="Email Confirmation Image"></a>
-            </div>
-            <h2 class="section-heading">2. Confirmation</h2>
-            <p>We will email you all the info you need to get started.</p>
-          </div>
-          <div class="col-3">
-            <div class="">
-              <img class="non-profit-icons" src="img/icons/project-starts.svg" alt="Email Confirmation Image"></a>
-            </div>
-            <h2 class="section-heading">3. Done!</h2>
-            <p>Choose your project and get hacking!</p>
-          </div>
+      <div class="col-3">
+        <div class="">
+          <img class="non-profit-icons" src="img/icons/email-confirmation.svg" alt="Email Confirmation Image"></a>
```

Comment:
Could you double what's going on with the `<img>` closing tags? They're all `</a>`